Makefile

```diff
@@ -442,11 +443,16 @@ endif
 #$(DATA_INSTALL_DIR) :
 #	$(MKDIR_PATH) $@
 
+# set setuid bit of extract_plugin_snapshot
+setuid_extract_plugin_snapshot:
+	chmod u+s $(INSTALL_DIR)/extras/C/extract_plugin_snapshot
```

Comment:
`chmod u+s $(EXTRAS_INSTALL_DIR)/C/extract_plugin_snapshot`

Reply:
@ashish-jabble can you please elaborate upon the shortcomings/errors in the current code and how the proposed code solves/fixes that?

Reply:
Reuse `$(EXTRAS_INSTALL_DIR)` it. In future if we want to change the directory structure so we need to change at one place NOT on basis of find occurrence.